site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -2,9 +2,191 @@
 api: scripting
 ---
 
+## Manifest
+
+In order to use the `chrome.scripting` API, you need to specify a
+`"manifest_version"` of `3` or higher and include the `"scripting"` permission
+in your [manifest file](manifest).
+
+```json
+{
+  "name": "Scripting Extension",
+  "manifest_version": 3,
+  "permissions": ["scripting"],
+  ...
+}
+```
+
+## Usage
+
+You can use the `chrome.scripting` API to inject JavaScript and CSS into
+websites, similar to what can be achieved with
+[content scripts](contentScripts). With the `chrome.scripting` API, extensions
+can make decisions at runtime.
+
+### Injection Targets
+
+You can use the `InjectionTarget` to specify an appropriate target to inject
+JavaScript or CSS into.
+
+The only required field is `tabId`. By default, an injection will run in the
+main frame of the specified tab.
+
+```js
+const tabId = getTabId();
+chrome.scripting.executeScript(
+    {
+      target: {tabId: tabId},
+      files: ['script.js'],
+    },
+    () => { ... });
+```
+
+To run in all frames of the specified tab, you can set the `allFrames` boolean
+to `true`.
+
+```js
+const tabId = getTabId();
+chrome.scripting.executeScript(
+    {
+      target: {tabId: tabId, allFrames: true},
+      files: ['script.js'],
+    },
+    () => { ... });
+```
+
+You can also inject in specific frames of a tab by specifying individual frame
+IDs. For more information on frame IDs, see the
+[webNavigation API](webNavigation).
+
+```js
+const tabId = getTabId();
+const frameIds = [frameId1, frameId2];
+chrome.scripting.executeScript(
+    {
+      target: {tabId: tabId, frameIds: frameIds},
+      files: ['script.js'],
+    },
+    () => { ... });
+```
+
+Note that you cannot specify both the `frameIds` and `allFrames` properties.
+
+### Injected Code
+
+Extensions can specify the code to be injected either via an external file or a
+runtime variable.
+
+#### Files
+
+Files are specified as strings that are paths relative to the extension's root
+directory. The following code will inject the file `script.js` into the main
+frame of the tab.
+
+```js
+const tabId = getTabId();
+chrome.scripting.executeScript(
+    {
+      target: {tabId: tabId},
+      files: ['script.js'],
+    },
+    () => { ... });
+```
+
 !!!.aside.warning
-This documentation is in active development.
-We're planning to expand the introduction section to provide more context and guidance on how to use this API.
-This work is tracked in issue [#194](https://github.com/GoogleChrome/developer.chrome.com/issues/194).
+Note: Currently, a maximum of a single file is supported.
```

Comment:
Either:
!!!.aside.aside--warning //This is the syntax you want, but warning is too strong
Warning: Currently...

Or:
!!!.aside
Note: Currently...

Reply:
Updated to !!!.aside